src/pages/proposals/ProposalList.vue

```diff
@@ -475,12 +497,46 @@ q-page.page-proposals
       :viewSelectorLabel="'View'",
       :chipsFiltersLabel="'Proposal types'",
       :filters.sync="filters"
-      :toggleLabel="'Show Staging Proposals'"
+      :toggleLabel="'Staging Proposals'"
       :toggle.sync="showStagedProposals",
       :toggleDefault="true"
       :showToggle="true",
       :style="mobileFilterStyles"
+      :filterTitle="'Search proposals'"
+      :showViewSelector="false"
       )
+      .col
+        base-placeholder.q-mr-sm(
+          v-if="!filteredProposals.length && !filteredStagedProposals.length && !$apollo.loading"
+          title= "No Proposals"
+          subtitle="Your organization has not created any proposals yet. You can create a new proposal by clicking the button below."
+          icon= "fas fa-file-medical"
+          :actionButtons="[{label: 'Create a new Proposal', color: 'primary', onClick: () => $router.push(`/${this.daoSettings.url}/proposals/create`), disable: !isMember, disableTooltip: 'You must be a member'}]"
```

Comment:
@evgenibir  would be nicer to have `onClick: handleCreateNewProposal` and above in JS to have method `handleCreateNewProposal`. This inlined method can become messy pretty easily